tests/apichecks/testdata/exceptions/outputonlyspecfields.txt

```diff
@@ -1,3 +1,5 @@
+[output_only_spec_field] crd=containerclusters.container.cnrm.cloud.google.com version=v1beta1: field ".spec.clusterAutoscaling.autoProvisioningDefaults.management.upgradeOptions[].autoUpgradeStartTime" is not set in unstructured objects
```

Comment:
we're only adding to this file for backwards compatibility right? As in, with the changes in this PR, these fields will never get set, right?

Reply:
I think so. Basically, these fields were placed under spec by mistake, and cannot be covered by tests, but we must keep them under spec for backwards compatibility purposes. So I added this exception file to track such edge cases.